docs/readme-mynewt.md

```diff
@@ -40,10 +40,11 @@ in this new format. That is done by passing the extra parameter `-2` as in:
 # Boot serial functionality with Mynewt
 
 Building with `BOOT_SERIAL: 1` enables some basic management functionality
-like listing images and uploading a new image to `slot0`. The serial bootloader
-requires that `mtu` is set to a value that is less than or equal to `256`.
-This can be done either by editing `~/.newtmgr.cp.json` and setting the `mtu`
-for the connection profile, or specifying you connection string manually as in:
+like listing images and uploading a new image to `primary slot`. The serial
```

Comment:
I think it is a bit weird to use this terminology here, because all Mynewt docs use slot0 and slot1, so will be confusing for uses; at least it should be mentioned somewhere that terminology was/will be changed

Reply:
This is also one of a handful that are still ungrammatical after the change.